Jenkinsfile

```diff
@@ -215,6 +216,7 @@ pipeline {
                         stage('Test') {
                             environment {
                                 SUREFIRE_REPORT_NAME_SUFFIX = 'it-testrun'
+                                LIMITS_CPU = 2
```

Comment:
Let's also set `FORK_COUNT = 2 ` or so, currently this PR seems to slow down the IT by a lot.

Reply:
I'm confused how it could affect it at all - there's only two modules, so lowering the limits here shouldn't matter - it can still only process two modules in parallel :thinking:

Reply:
It takes 20 minutes to process the IT stage because the update tests take 20 minutes. This is independent of this PR :slightly_smiling_face:

Reply:
Ah, I missed that. But that's totally fine then. I didn't consider that the default fork count from the pom would be used. Let's leave it like this.